lib/ivis_opengl/piedraw.cpp

```diff
@@ -297,11 +298,191 @@ namespace
 
 		~glBufferWrapper()
 		{
-			glDeleteBuffers(1, &id);
+			if (id != 0)
```

Comment:
This is unnecessary. glDeleteBuffers() called on a zero value ID is a no-op according to the standard. Same in similar cases below.